saos-webapp/src/main/webapp/WEB-INF/view/details/judgmentDetailsMeta.jsp

```diff
@@ -0,0 +1,176 @@
+    </c:forEach>
+    
+    <c:set var="detailsMetaKeywords" value="${detailsMetaKeywords}, ${metaJudges}" />
+</c:if>
+
+<%-- Keywords --%> 
+<c:if test="${!empty judgment.keywords}" >
+
+    <c:set var="metaKeywords" value="" ></c:set>
+    <c:forEach items="${judgment.keywords}" var="keyword" varStatus="status" >
+        
+        <c:set var="metaKeywords" value="${metaKeywords}${keyword.phrase }" />
+        
+        <c:if test="${!status.last}">
+            <c:set var="metaKeywords" value="${metaKeywords}, " />
+        </c:if>
+    </c:forEach>
+    
+    <c:set var="detailsMetaKeywords" value="${detailsMetaKeywords}, ${metaKeywords}" />
+</c:if>
+
+<%-- LegalBases --%> 
+<c:if test="${!empty judgment.legalBases}" >
+
+    <c:set var="metaLegalBases" value="" ></c:set>
+    <c:forEach items="${judgment.legalBases}" var="legalBase" varStatus="status" >
+        
+        <c:set var="metaLegalBases" value="${metaLegalBases}${legalBase }" />
+        
+        <c:if test="${!status.last}">
+            <c:set var="metaLegalBases" value="${metaLegalBases}, " />
+        </c:if>
+    </c:forEach>
+    
+    <c:set var="detailsMetaKeywords" value="${detailsMetaKeywords}, ${metaLegalBases}" />
+</c:if>
+
+
+<meta name="keywords" content="${detailsMetaKeywords}" />
+
+
+<%-- Meta description --%> 
+<spring:message var="detailsMetaDescription" code="details.meta.judgment" />
+
+<c:choose>
+    <c:when test="${judgment.judgmentType == 'RESOLUTION' }" >
+        <spring:message var="releasedBy" code="details.meta.releasedBy.feminine" />
+    </c:when>
+    <c:when test="${judgment.judgmentType == 'SENTENCE' }" >
+        <spring:message var="releasedBy" code="details.meta.releasedBy.masculine" />
+    </c:when>
+    <c:otherwise>
+        <spring:message var="releasedBy" code="details.meta.releasedBy.neuter" />
+    </c:otherwise>
+</c:choose>
+
+
+<c:if test="${!empty metaJudgmentType}" >
+    <c:set var="detailsMetaDescription" value="${metaJudgmentType}" />
+</c:if>
+
+<c:if test="${!empty metaCaseNumbers}" >
+    <spring:message code="details.meta.withSignature" var="withSignature" /> 
+    <c:set var="detailsMetaDescription" value="${detailsMetaDescription} ${withSignature} ${metaCaseNumbers}" />
+</c:if>
+
+<c:choose>
+    <c:when test="${judgment.courtType == 'COMMON'}" >
+        <c:if test="${!empty judgment.courtDivision.court.name}" >
+            <c:set var="detailsMetaDescription" value="${detailsMetaDescription} ${releasedBy} ${judgment.courtDivision.court.name}" />
+        </c:if>
+    </c:when>
+    <c:when test="${judgment.courtType == 'SUPREME'}" >
+        <c:if test="${!empty metaScChambers}" >
+            <c:set var="detailsMetaDescription" value="${detailsMetaDescription} ${releasedBy} ${metaScChambers}" />
+        </c:if>
+    </c:when>
+    <c:otherwise>
+        <c:if test="${!empty metaCourtType}" >
+		    <c:set var="detailsMetaDescription" value="${detailsMetaDescription} ${releasedBy} ${metaCourtType}" />
```

Comment:
Postanowienie o sygnaturze KIO 225/13 wydane przez Krajowa Izba Odwoławcza w składzie sędziowskim Katarzyna Prowadzisz.
should be
Postanowienie o sygnaturze KIO 225/13 wydane przez Krajową Izbę Odwoławczą w składzie sędziowskim: Katarzyna Prowadzisz.